README.md

```diff
@@ -86,17 +86,24 @@ containers to already be running.
 
 ### `make routes`
 
-Displays all the application's HTTP routes.
+Displays all the application’s HTTP routes.
+
+
+## Testing
+
+We currently have a shortage of test coverage. However, we are working on 
+addressing this. We welcome and appreciate any contributions you can make to 
+assist with this task.
 
 
 ## Contributing
 
-Before committing changes, you can lint and reformat your code using
-[tox](https://tox.wiki/). To install this tool on your local machine or a
-virtual environment:
+Thank you very much for your interest! We have a guide designed for you, 
+please take a moment to read it at [CONTRIBUTING.md](./CONTRIBUTING.md).
```

Comment:
Ah I see! 
Mh, but doesn't linting belong to development workflow?

Reply:
For me is more related with the contributing process...?

Reply:
Dunno